hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestFileUtil.java

```diff
@@ -726,7 +727,10 @@ public void testUnZip() throws IOException {
     // check result:
     assertTrue(new File(tmp, "foo").exists());
     assertEquals(12, new File(tmp, "foo").length());
-    
+    assertTrue(new File(tmp, "foo").canExecute());
```

Comment:
asserttrue tests need to say what is wrong when the assert fails. the goal is to make a failed yetus/test run meaningful.

Reply:
can you add ext to the asserts for permissons, e.g

```java
assertTrue("file lacks execute permissions", new File(tmp, "foo").canExecute());
```

imagine a yetus test run failed? what do you want in the test reports. a line number or a message?

Reply:
That would be a clearer explanation.I will add it.